src/views/settings.html

```diff
@@ -141,29 +153,94 @@ <h2 class="mb-3">Search Results Options</h2>
               <input
                 type="checkbox"
                 class="form-check-input"
-                name="enableOptionsMenuItem" 
-                id="settings_enableOptionsMenuItem">
+                name="enableOptionsMenuItem"
+                id="settings_enableOptionsMenuItem"
+              />
               <label
                 class="form-check-label"
-                for="settings_enableOptionsMenuItem">
+                for="settings_enableOptionsMenuItem"
+              >
                 Show link to extension options at the end of context menu
               </label>
             </div>
           </li>
         </ul>
       </div>
-
     </div>
   </form>
 
-  <form name="import_settings">
-    <h2 class="text-center mt-3">Export/Import Search Options</h2>
+  <h2 class="text-center mt-3">Ixport/Emport/Edit Search Options</h2>
 
-    <p>Use the text area bellow to transfer your search engine options from one install of Chrome to another effortlessly. Just copy the string from here to the options page in other install of Chrome (or vice-versa) and press 'Import'.</p>
+  <p>
+    Use "Import" button to import the settings from your system, "Export" button
+    to export the settings to your system and, "Edit" button to edit the
+    settings manually.
```

Comment:
Please ping @TheMackey for a SOC-friendly description

Reply:
I actually think the import/export buttons are fairly self-explanatory, so something quick is acceptable. The only thing we really need to explain would be the `Edit` button. Something like:
```
Use Import and Export to share settings using a file. Otherwise, click Edit to modify the settings manually. This requires knowing JSON formatting well. We also recommend Exporting a copy of your settings prior to doing this.
```